app/moves/middleware.test.js

```diff
@@ -35,32 +35,107 @@ describe('Moves middleware', function() {
     })
 
     context('with current location', function() {
-      const mockLocationId = 'c249ed09-0cd5-4f52-8aee-0506e2dc7579'
+      context(
+        "when user hasn't got permission to see the proposed moves",
+        function() {
+          context('when location type is police', function() {
```

Comment:
Not sure you fully understood what I meant. I think the tests were fine before, it was just the description in the `context` block.

The current method has zero concern with location type, so there is no need to have tests with different location types, they'll function the same. I was just pointing out the wording was just from the previous implementation.

These two tests don't offer anything different when testing this method.

Reply:
Once I had changed the wording of the context, it made more sense to reorganise it this way.

Reply:
Following our conversation, I have now removed the redundant test.